.github/workflows/dest_mysql.yml

```diff
@@ -24,19 +24,6 @@ jobs:
     defaults:
       run:
         working-directory: ./plugins/destination/mysql
-    services:
```

Comment:
Any specific reason for this?

Reply:
Maps and Lists don't work without it, see https://github.com/cloudquery/cloudquery/pull/13557/commits/a429640d33d67750b90d0ff8f1c630ee4c8fb16f

Reply:
Ah, OK.
Could you add a comment to the workflow describing this? So that we won't try to move back to `services` in this workflow.

Reply:
There's a comment there `# We don't use services as we need to pass additional arguments to the container`